### PR TITLE
Reviewing SotA

### PR DESCRIPTION
**Pull request by fbarranco**

### CODE REVIEW

File: edubots_surveys.Rnw [hunk 1]
```diff
@@ -317,17 +317,17 @@ The widespread and rapid adoption of free Mobile Instant Messaging
 (MIM) tools/platforms such as WhatsApp, Telegram, WeChat and Facebook
 Messenger stems from their simplicity, ease of use and multi-modality
 (i.e. video, audio, text) \cite{tang2017mobile}. Using these tools in
-higher education can facilitate the delivery of personalized learning
-that occurs anytime anywhere, promote collaborative learning
+higher education facilitates the delivery of personalized learning
+that occurs anytime anywhere, and promotes collaborative learning
```
Comment (JJ) [inline edubots_surveys.Rnw:318]:
```suggestion
that occurs anytime anywhere and promotes collaborative learning
```

File: edubots_surveys.Rnw [hunk 2]
```diff
@@ -317,17 +317,17 @@ The widespread and rapid adoption of free Mobile Instant Messaging
...
 experiences and group discussions \cite{panah2020study}.
 
 WhatsApp is, at least in most Western countries, the most popular MIM platform
-used by educators to \textcolor{black}{provide} assignments' feedback to students, support course
-discussions, and provide learning resources in informal learning settings
+used by educators to provide assignments' feedback to students, support course
```
Comment (JJ) [inline edubots_surveys.Rnw:322]:
```suggestion
which is why it is the tool preferably used by educators to provide assignments' feedback to students, support course
```

File: edubots_surveys.Rnw [hunk 3]
```diff
@@ -317,17 +317,17 @@ The widespread and rapid adoption of free Mobile Instant Messaging
...
+discussions, and contribute resources in informal learning settings
```
Comment (JJ) [inline edubots_surveys.Rnw:323]:
```suggestion
discussions, and post learning resources in a semi-formal learning context
```

File: edubots_surveys.Rnw [hunk 4]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
 communities for students \cite{abbas2021onlinechat}. However, these rules should
 not affect students' ownership and control, since it is vital to advance in
 their learning \cite{gachago2015crossing} process.  This is why examining the
-role of MIM in higher education is still a challenge, and why the opinions of
+role of MIM platforms in higher education is still a challenge, and why the opinions of
```
Comment (JJ) [inline edubots_surveys.Rnw:341]:
```suggestion
role of MIM platforms in higher education, as well as applications based on them, such as chatbots,  is still a challenge, and why the opinions of
```

File: edubots_surveys.Rnw [hunk 5]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
 the teaching community towards them have to be examined, as we do in this paper.
 
 % The Use of Chatbots in Higher Education
-The use of MIM, although possibly valuable by itself, can be enhanced via the
-use of chatbots, which, being conversational agents, usually dwell in systems
-where synchronous conversations take place.  The use of conversational agents
-(chatbots) in higher education is still at its infancy
-\cite{yang2019opportunities}. Nevertheless, recent studies examining their
+The use of MIM platforms, although possibly valuable by itself, can be enhanced with the
+use of chatbots, which are conversational agents that normally dwell
```
Comment (JJ) [inline edubots_surveys.Rnw:346]:
```suggestion
use of chatbots, which are conversational agents that normally dwell in
```

File: edubots_surveys.Rnw [hunk 6]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-(chatbots) in higher education is still at its infancy
-\cite{yang2019opportunities}. Nevertheless, recent studies examining their
+The use of MIM platforms, although possibly valuable by itself, can be enhanced with the
+use of chatbots, which are conversational agents that normally dwell 
+synchronous conversations systems. The use of chatbots in higher education is still in its early stages
+\cite{yang2019opportunities}. Nevertheless, recent studies have shown their
 positive impact on students' academic performance \cite{perez2020rediscovering}
-and engagement \cite{differ.chat,abbas2021onlinechat} have led to a growing
+and engagement \cite{differ.chat,abbas2021onlinechat}, leading to a growing
 interest in using this technology in the (possibly virtual) classroom. Indeed,
```
Comment (JJ) [inline edubots_surveys.Rnw:351]: by whom?

File: edubots_surveys.Rnw [hunk 7]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-the use of chatbots in collecting course feedback from students in higher
-education improved students' response quality and boosted their enjoyment levels
-\cite{abbas2021Surveys}.  According to \cite{Roblyer2010}, using either tools
-such as mobile devices or teaching strategies based on gamification
+the using chatbots to collect course feedback from students in higher
```
Comment (JJ) [inline edubots_surveys.Rnw:352]:
```suggestion
using chatbots to collect course feedback from students in higher
```

File: edubots_surveys.Rnw [hunk 8]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+education improved response quality and boosted enjoyment levels
```
Comment (JJ) [inline edubots_surveys.Rnw:353]:
```suggestion
education improved the quality of response given by students in their assignments, and boosted engagement levels
```

File: edubots_surveys.Rnw [hunk 9]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+\cite{abbas2021Surveys}.  According to \cite{Roblyer2010}, the use of
+mobile devices or teaching strategies based on gamification
```
Comment (JJ) [inline edubots_surveys.Rnw]: or → along with? Based on?

File: edubots_surveys.Rnw [hunk 10]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-Messenger with the focus to identify characteristics and quality metrics such as
-language, subject matter and platform\textcolor{black}{, whereas the study undertaken by Pérez and collaborators
-\cite{perez2020rediscovering} aimed to categorize educational chatbots, according
-to their, purpose into service-oriented and teaching-oriented}. The first category
-includes those that provide service support such as the chatbot Ask Holly
-\cite{meetholly} and Dina \cite{santoso2018dinus}; both chatbots respond to
+students' knowledge and mitigating their feelings of isolation.
+
+Several studies have been conducted to evaluate chatbots in higher education. For
+instance, a recent study was presented by Smutny and Schreiberova
```
Comment (JJ) [inline edubots_surveys.Rnw]: not so recent anymore? First version of this paper was late 2021...

File: edubots_surveys.Rnw [hunk 11]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-proposed a generic chatbot for university students that is able to answer to
-their frequently asked questions \textcolor{black}{and, for some time, the University of Granada had
-\textcolor{black}{Elvira} as a chatbot in its main web page \cite{MOREO20129} to perform that duty.} Besides being able
-to answer pre-established (frequent) questions, it did so from the website of
-the University of Granada using an inset \textcolor{black}{persona, voiced by a real person, who
-lip-synced the answers. The emphasis in the case of Elvira, however, was on the
-authenticity of the synchronization more than in making the update of questions
-answered simpler, or more interactive, or more open to the rest of the university staff, using actual messaging platforms. In
+proposed a generic chatbot for university students that is able to answer 
+frequently asked questions. The University of Granada had
```
Comment (JJ) [inline edubots_surveys.Rnw:376]:
```suggestion
frequently asked questions. The University of Granada officially deployed
```

File: edubots_surveys.Rnw [hunk 12]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-by hand (and not too often). It was eventually discontinued, and its technology was not adapted to other platforms, since it did not really provide an useful service. Its service was not tied to teaching anyway; it was more related to directory queries and university-wide administrative questions, so even if it is strictly an example of a chatbot in a university setting, it is not really a chatbot that can be adopted by teachers to affect the learning outcomes, something which is the focus of this paper}.
+and accurate answers than those provided by Elvira, which had to be updated
+by hand (and not too often). It was eventually discontinued, and its technology was not adapted to other platforms, since it did not really provide a useful service. Its service was not tied to teaching anyway; it was more related to directory queries and university-wide administrative questions, so even if it is strictly an example of a chatbot in a university setting, it is not really a chatbot that can be adopted by teachers to affect the learning outcomes, which is the focus of this paper.
 
 On the other hand, teaching-oriented chatbots are more sophisticated, as they
```
Comment (JJ) [inline edubots_surveys.Rnw]: more sophisticated than what?

File: edubots_surveys.Rnw [hunk 13]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+\cite{sjostrom2018designing,tsivitanidouusers}. These needs include, but are not limited to, student learning success due to higher motivation, but these are a posteriori effects that cannot be assessed in advance. This is why we focus in this paper on what  teachers are looking for in a Technology Acceptance Model (TAM) and associated technologies like chatbots. Both authors focus on what teachers need in terms of the latter; this paper will focus on a wider perspective, trying to determine what they are looking for in terms of general messaging technology. The classical literature \cite{tam91} already proposes that any adoption of technology must be tuned to the user needs and experience. In this paper, we will try to find out what those are in order to propose a successful model of adoption of chatbot technology.
```
Comment (JJ) [inline edubots_surveys.Rnw]: success is not really a need, but an objective, right? Maybe we can rephrase this in some other way?

File: edubots_surveys.Rnw [hunk 14]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
 students in a Chinese class \cite{doi:10.1177/0735633120929622}. This paper
-analysed conversational chatbots in an one-to-one setting, finding that learning
-really benefited from it. This is, once again, a proof of the benefit of
-chatbots in certain settings; however, there are some prior experiences, as well as needs, that might
+analysed conversational chatbots in a one-to-one setting, finding that learning
+truly benefited from it. This is, once again, an evidence of the benefit of
+chatbots in certain settings; however, there are some prior experiences, as well as needs, that may
 prevent the successful acceptance of the technology, and these are what we are
-trying to find out in this paper, together with what kind of features would improve
+trying to find out in this paper, alng with what kind of features would improve
```
Comment (JJ) [inline edubots_surveys.Rnw:446]:
```suggestion
trying to find out in this paper, along with what kind of features would improve
```

**Review by JJ — CHANGES_REQUESTED** [review]
Mostly OK, please consider these suggestions.

File: edubots_surveys.Rnw (hunk 12 above)
Comment (fbarranco) [reply edubots_surveys.Rnw]: I guess we meant more sophisticated than the chatbots discussed in the previous paragraph, that were focused on administrative tasks.

File: edubots_surveys.Rnw (hunk 9 above)
Comment (fbarranco) [inline edubots_surveys.Rnw]:
```suggestion
mobile devices along with gamification strategies
```

File: edubots_surveys.Rnw (hunk 12 above)
Comment (JJ) [reply edubots_surveys.Rnw]: Can you please add that to the text?

File: edubots_surveys.Rnw [hunk 15]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-by hand (and not too often). It was eventually discontinued, and its technology was not adapted to other platforms, since it did not really provide an useful service. Its service was not tied to teaching anyway; it was more related to directory queries and university-wide administrative questions, so even if it is strictly an example of a chatbot in a university setting, it is not really a chatbot that can be adopted by teachers to affect the learning outcomes, something which is the focus of this paper}.
+and accurate answers than those provided by Elvira, which had to be updated
+by hand (and not too often). It was eventually discontinued, and its technology was not adapted to other platforms, since it did not really provide a useful service. Its service was not tied to teaching anyway; it was more related to directory queries and university-wide administrative questions, so even if it is strictly an example of a chatbot in a university setting, it is not really a chatbot that can be adopted by teachers to affect the learning outcomes, which is the focus of this paper.
 
 On the other hand, teaching-oriented chatbots are more sophisticated, as they
```
Comment (fbarranco) [inline edubots_surveys.Rnw]:
```suggestion
Contrarily to the mentioned chatbots that are focused on administrative tasks, teaching-oriented chatbots are more sophisticated, as they
```

File: edubots_surveys.Rnw [hunk 16]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
-Messenger with the focus to identify characteristics and quality metrics such as
-language, subject matter and platform\textcolor{black}{, whereas the study undertaken by Pérez and collaborators
-\cite{perez2020rediscovering} aimed to categorize educational chatbots, according
-to their, purpose into service-oriented and teaching-oriented}. The first category
-includes those that provide service support such as the chatbot Ask Holly
-\cite{meetholly} and Dina \cite{santoso2018dinus}; both chatbots respond to
+students' knowledge and mitigating their feelings of isolation.
+
+Several studies have been conducted to evaluate chatbots in higher education. For
+instance, a recent study was presented by Smutny and Schreiberova
```
Comment (fbarranco) [inline edubots_surveys.Rnw]:
```suggestion
instance, a thorough study was presented by Smutny and Schreiberova
```

File: edubots_surveys.Rnw [hunk 17]
```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
...
+\cite{sjostrom2018designing,tsivitanidouusers}. These needs include, but are not limited to, student learning success due to higher motivation, but these are a posteriori effects that cannot be assessed in advance. This is why we focus in this paper on what  teachers are looking for in a Technology Acceptance Model (TAM) and associated technologies like chatbots. Both authors focus on what teachers need in terms of the latter; this paper will focus on a wider perspective, trying to determine what they are looking for in terms of general messaging technology. The classical literature \cite{tam91} already proposes that any adoption of technology must be tuned to the user needs and experience. In this paper, we will try to find out what those are in order to propose a successful model of adoption of chatbot technology.
```
Comment (fbarranco) [inline edubots_surveys.Rnw]:
```suggestion
\cite{sjostrom2018designing,tsivitanidouusers}. These needs include, but are not limited to, student learning success due to higher motivation, but these are a posteriori effects that cannot be assessed in advance. This is why we focus in this paper on what  teachers are looking for in a Technology Acceptance Model (TAM) and associated technologies like chatbots. Both authors focus on what teachers need in terms of the latter; this paper will focus on a wider perspective, trying to determine what they are looking for in terms of general messaging technology. The classical literature \cite{tam91} already proposes that any adoption of technology must be tuned to the user needs and experience. In this paper, we will try to find out what those are in order to propose a model to facilitate the adoption of chatbot technology.
```

**Review by JJ — APPROVED** [review]
:+1: LGTM

**Pull request merged**